lib/app/components/pages/dashboard/controllers/dashboard_controller.dart

```diff
@@ -41,4 +67,14 @@ class DashboardController extends GetxController {
       _ad!.dispose();
     }
   }
+
+  void changeIndex(int index) {
+    // モーダルの場合
+    if(navigationInfo[index].isModal){
+      Get.dialog(navigationInfo[index].modalWindow!); // ダイアログで表示
+    }
+    else{
+      tabService.changeIndex(index);  // インデックスを変更
+    }
+  }
```

Comment:
```suggestion
  void changeIndex(int index) {
      tabService.changeIndex(index); 
  }
```
ごめんなさい。前の実装に戻して欲しいです。🙇